src/flexalloc_shared.h

```diff
@@ -7,6 +7,7 @@ extern "C" {
 #endif
 
 #define FLA_ERR_ERROR 1001
+#define FLA_ERR_ALL_SLABS_USED 2001
```

Comment:
Where did 1001 and 2001 come from? I was wondering since the error code is passed around as an int